smartcar/smartcar.py

```diff
@@ -65,16 +65,26 @@ def __init__(self, client_id, client_secret, redirect_uri, scope=None, developme
                 or declines the application's permissions. This URL must also be
                 present in the Redirect URIs field in the application dashboard
             scope (bool, optional): A list of permissions requested by the application
-            development (bool, optional): If True, deplays the Mock OEM for testing.
-                Defaults to False
+            test_mode (bool, optional): Launch the Smartcar auth flow in test mode. Defaults to false.
+                https://smartcar.com/docs#request-authorization
+            development (bool, optional): DEPRECATED Launch the Smartcar auth flow in development mode
+                to enable mock vehicle brands.
 
         """
         self.client_id = client_id
         self.client_secret = client_secret
         self.auth=(client_id, client_secret)
         self.redirect_uri = redirect_uri
         self.scope = scope
-        self.development = development
+
+        if development:
+            import warnings
```

Comment:
Is it best to put imports nested in in methods like this?

Reply:
discussed irl, pythonic

Reply:
modules are cached